Apps/WebClient/src/ClientApp/app/components/timeline/healthlink.vue

```diff
@@ -0,0 +1,88 @@
+<style lang="scss" scoped>
+.box {
+  box-shadow: 0 2.8px 2.2px rgba(0, 0, 0, 0.015),
```

Comment:
bootstrap has shadow classes, lets use those instead
https://getbootstrap.com/docs/4.4/utilities/shadows/

Reply:
revised to bootstrap default